templates/index.html

```diff
@@ -332,8 +339,51 @@
                   });
             });
 
+            $('#emailAcknowledgementBtn').on('click', function() {
+                var emails = [];
+                var table = document.getElementById('studentTable');
+                var checkboxes = table.getElementsByClassName('studentCheckbox');
+
+                for (var i = 0; i < checkboxes.length; i++) {
+                    if (checkboxes[i].checked) {
+                        var row = checkboxes[i].closest('tr');
+                        var email = row.cells[4].innerText;
+                        if (email) {
+                            emails.push(email);
+                        }
+                    }
+                }
+
+                if (emails.length > 0) {
+                    var emailString = emails.join(',');
+                    var mailtoLink = "mailto:?subject=Acknowledgement of your application to the WEHI student internship program&body=Hi All,%0D%0A%0D%0AThank you for applying for the WEHI Student Internship program.%0D%0A%0D%0AI will be calling you by mobile phone over the next week to get a better understanding of projects that fit your interests and career aspirations. Please let me know if you need to make other arrangements as I can be flexible.%0D%0A%0D%0AYou can see a list of projects as well as other details on the WEHI Student Internship page, where you can click on the project to go to a project page.%0D%0A%0D%0AEach project page has a rough description of the role outline, but the key information is the 'Further Reading' section that may be available for that project.%0D%0A%0D%0APlease also be acquainted with the program handbook.%0D%0A%0D%0ADetails of duties are available via the student projects page at https://wehi-researchcomputing.github.io/students.%0D%0A%0D%0ADetails of the intake dates are available at https://wehi-researchcomputing.github.io/intake_dates.%0D%0A%0D%0AA FAQ is also available at https://wehi-researchcomputing.github.io/faq" + "&to=" + emailString;
```

Comment:
can possibly take this string and assign it to a separate variable and store it there to reduce congestion within the function